application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java

```diff
@@ -45,7 +53,13 @@ public void onMessageReceived(MessageReceivedEvent event) {
         }
 
         if (messageHasNoMediaAttached(message)) {
-            message.delete().flatMap(any -> dmUser(message)).queue();
+            message.delete().flatMap(any -> dmUser(message)).queue(res -> {
+            }, new ErrorHandler().handle(ErrorResponse.CANNOT_SEND_TO_USER, err -> {
+                warnUser(message).queue((res) -> {
+                    res.delete().queueAfter(1, TimeUnit.MINUTES);
+                }, (error) -> {
+                });
```

Comment:
u can remove the error part entirely, just keep the first part

Reply:
sure I will do it